### PR TITLE
#3292 recovery fix

### PR DESCRIPTION
**Pull request by AlexeyTuboltsev**

### CODE REVIEW

File: packages/web/app/components/layouts/Layout.tsx [hunk 1]
```diff
@@ -87,8 +87,19 @@ const Layout = compose<IStateProps, TDataTestId & TContentExternalProps & ILayou
   }),
 )(LayoutComponent);
 
+const TransitionalLayout: React.FunctionComponent<
```
Comment (desfero) [inline packages/web/app/components/layouts/Layout.tsx]: +1

File: packages/web/app/components/layouts/header/Header.tsx [hunk 2]
```diff
@@ -100,17 +83,4 @@ export const HeaderAuthorized: React.FunctionComponent<{}> = () => (
   </div>
 );
 
-export const HeaderUnauthorized = compose<
-  IHeaderUnauthProps & IHeaderUnauthExternalProps,
-  IHeaderUnauthExternalProps
->(
-  appConnect<IHeaderUnauthProps, {}>({
-    stateToProps: s => ({
-      isIssuerLocation: Boolean(
```
Comment (desfero) [inline packages/web/app/components/layouts/header/Header.tsx]: this change will introduce a bug. Go to `eto-landing` page then click `Get started` and fill all details to create an account, after that you are registered on the platform as an investor which is incorrect.

File: packages/web/app/test-e2e/wallets/WalletRecover.spec.e2e.io.ts [hunk 3]
```diff
@@ -10,24 +12,23 @@ import {
...
     cyPromise(() => generateRandomSeedAndAddress("m/44'/60'/0'")).then(
       ({ seed: words, address: expectedGeneratedAddress }) => {
+        const password = "strongpassword";
         const email = generateRandomEmailAddress();
 
         cy.visit(`${recoverRoutes.seed}`);
 
         typeLightwalletRecoveryPhrase(words);
 
         cy.get(tid("wallet-selector-register-email")).type(email);
```
Comment (desfero) [inline packages/web/app/test-e2e/wallets/WalletRecover.spec.e2e.io.ts]: Probably we can replace this by `typeEmailPassword` util

File: packages/web/app/test-e2e/wallets/WalletRecover.spec.e2e.io.ts [hunk 4]
```diff
@@ -10,24 +12,23 @@ import {
...
 
 describe("Wallet recover", function(): void {
   this.retries(2);
   it("should recover wallet from saved phrases", () => {
     cyPromise(() => generateRandomSeedAndAddress("m/44'/60'/0'")).then(
       ({ seed: words, address: expectedGeneratedAddress }) => {
+        const password = "strongpassword";
         const email = generateRandomEmailAddress();
 
         cy.visit(`${recoverRoutes.seed}`);
```
Comment (desfero) [inline packages/web/app/test-e2e/wallets/WalletRecover.spec.e2e.io.ts]: No need for a template literal here

File: packages/web/app/components/layouts/header/Header.tsx (hunk 2 above)
Comment (AlexeyTuboltsev) [reply packages/web/app/components/layouts/header/Header.tsx]: eto landing is not used anymore. I removed it and redirected eto-landing to normal landing. So it's fine

File: packages/web/app/test-e2e/wallets/WalletRecover.spec.e2e.io.ts (hunk 4 above)
Comment (AlexeyTuboltsev) [reply packages/web/app/test-e2e/wallets/WalletRecover.spec.e2e.io.ts]: 👍

File: packages/web/app/test-e2e/wallets/WalletRecover.spec.e2e.io.ts (hunk 3 above)
Comment (AlexeyTuboltsev) [reply packages/web/app/test-e2e/wallets/WalletRecover.spec.e2e.io.ts]: replaced all of them  with a util function

File: packages/web/app/modules/wallet-selector/actions.ts [hunk 5]
```diff
@@ -1,15 +1,17 @@
+import { createActionFactory } from "../../../../shared/dist/modules/actionsUtils";
```
Comment (desfero) [inline packages/web/app/modules/wallet-selector/actions.ts]: ups, import should be from `@neufund/shared`

File: packages/web/app/modules/wallet-selector/actions.ts [hunk 6]
```diff
@@ -1,15 +1,17 @@
...
 import { ledgerWizardActions } from "./ledger-wizard/actions";
 import { lightWizardActions } from "./light-wizard/actions";
 
 const actions = {
-  reset: () => createSimpleAction("WALLET_SELECTOR_RESET"),
-  connected: () => createSimpleAction("WALLET_SELECTOR_CONNECTED"),
-  messageSigning: () => createSimpleAction("WALLET_SELECTOR_MESSAGE_SIGNING"),
-  messageSigningError: (errorMessage: TMessage) =>
-    createAction("WALLET_SELECTOR_MESSAGE_SIGNING_ERROR", { errorMessage }),
+  reset: createActionFactory("WALLET_SELECTOR_RESET"),
```
Comment (desfero) [inline packages/web/app/modules/wallet-selector/actions.ts]: 🎖

File: packages/web/app/modules/wallet-selector/light-wizard/actions.ts [hunk 7]
```diff
@@ -1,15 +1,24 @@
...
-    createAction("LIGHT_WALLET_CONNECTION_ERROR", { errorMsg }),
-  lightWalletReset: () => createSimpleAction("LIGHT_WALLET_RESET"),
-  lightWalletLogin: (email: string, password: string) =>
-    createAction("LIGHT_WALLET_LOGIN", { email, password }),
-  lightWalletRecover: (email: string, password: string, seed: string) =>
-    createAction("LIGHT_WALLET_RECOVER", { email, password, seed }),
-  lightWalletBackedUp: () => createSimpleAction("LIGHT_WALLET_BACKUP"),
-  lightWalletRegister: (email: string, password: string) =>
-    createAction("LIGHT_WALLET_REGISTER", { email, password }),
+  lightWalletConnectionError: createActionFactory(
```
Comment (desfero) [inline packages/web/app/modules/wallet-selector/light-wizard/actions.ts]: 🎖

**Pull request merged**